composer.json

```diff
@@ -19,6 +19,7 @@
         "php": ">=5.5",
         "php-http/client-implementation": "^1.0",
         "php-http/message-factory": "^1.0.2",
+        "php-http/client-common": "^1.1",
         "php-http/plugins": "^1.0",
```

Comment:
ah sorry, i was confused. what you do is for the plugins that are now separate you use the new one if are available and otherwise keep using the old one. i think that is fine. once we move to 2.0 the separate plugins will only be available if included.